datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireFragment.kt

```diff
@@ -389,9 +389,9 @@ class QuestionnaireFragment : Fragment() {
      */
     internal const val EXTRA_QUESTIONNAIRE_RESPONSE_JSON_STRING = "questionnaire-response"
 
-    /** A JSON encoded string extra for questionnaire context. */
-    internal const val EXTRA_QUESTIONNAIRE_LAUNCH_CONTEXT_JSON_STRING =
-      "questionnaire-launch-context"
+    /** A list of JSON encoded strings extra for each questionnaire context. */
+    internal const val EXTRA_QUESTIONNAIRE_LIST_OF_LAUNCH_CONTEXTS_JSON_STRING =
+      "list-of-questionnaire-launch-contexts"
```

Comment:
optional : 
optional : 
```suggestion
    internal const val EXTRA_QUESTIONNAIRE_LAUNCH_CONTEXT_JSON_STRINGS =
      "questionnaire-launch-contexts"
```